scopes/harmony/aspect-loader/aspect-loader.cmd.ts

```diff
@@ -0,0 +1,30 @@
+import { AspectLoaderMain } from '@teambit/aspect-loader';
+import { Command, CommandOptions } from '@teambit/cli';
+import { CLITable } from '@teambit/cli-table';
```

Comment:
Does this one uses ink internally?
Is yes please replace it.
We have many (too many) tables which don't use ink.
Tty-table, table and more